data/ZTF21aaqjmps_phot.csv

```diff
@@ -0,0 +1,203 @@
+filter,mjd,flux,zp,magsys,fluxerr
```

Comment:
i-band data (ztfi) is probably still private, so please remove it from the commit.